app/main.py

```diff
@@ -1,3 +1,65 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from app.customer import Customer, Car
+from app.shop import Shop
+from os import path
+
+
+def load_data(file_name: str) -> tuple:
+    with open(file_name, "r") as file_in:
+        data = json.load(file_in)
+    customers = create_customers(data["customers"])
+    shops = create_shops(data["shops"])
+    fuel_price = data["FUEL_PRICE"]
+    return fuel_price, customers, shops
+
+
+def create_customers(data: dict) -> list[Customer]:
+    return [
+        Customer(
+            customer["name"],
+            customer["product_cart"],
+            customer["location"],
+            customer["money"],
+            Car(customer["car"]["brand"],
+                customer["car"]["fuel_consumption"])
+        )
+        for customer in data]
+
+
+def create_shops(data: dict) -> list[Shop]:
+    return [
+        Shop(
+            shop["name"],
+            shop["location"],
+            shop["products"])
+        for shop in data]
```

Comment:
```suggestion
def create_shops(data: dict) -> list[Shop]:
    return [Shop(**shop) for shop in data]
```
better to use unpacking in this case